src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -130,6 +131,39 @@ public override async Task<(IReadOnlyList<PackageSpec> dgSpecs, IReadOnlyList<IA
             return (new[] { packageSpec }, null);
         }
 
+        private async Task<bool> IsCentralPackageManagementVersionsEnabledAsync()
+        {
+            return MSBuildStringUtility.IsTrue(await _vsProjectAdapter.GetPropertyValueAsync(ProjectBuildProperties.ManagePackageVersionsCentrally));
+        }
+
+        private async Task<Dictionary<string, CentralPackageVersion>> GetCentralPackageVersionsAsync()
+        {
+            IEnumerable<(string PackageId, string Version)> packageVersions =
+                        (await _vsProjectAdapter.GetBuildItemInformationAsync("PackageVersion", "Version"))
```

Comment:
Both `PackageVersion` and `Version` are defined in `PackageBuildProperties`.

Reply:
Updated